duckduckgo/duckduckgo-privacy-extension
# Update Microsoft tracker icon

Blocked trackers are grouped by network in the extension popup
interface. Where possible an icon for the network is included. Update
the Microsoft icon, to make blocked Microsoft trackers easier to
identify.

Note: This does not change which trackers are blocked.

**Reviewer:** @GioSensation 

## Steps to test this PR:
1. Navigate to https://ballotpedia.org/Main_Page
2. Click the extension icon, then click "X Trackers blocked".
3. Scroll down on the list of blocked trackers. Ensure the Microsoft icon looks correct next to the blocked `clarity.ms` tracker.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
